runners/spark/src/main/java/org/apache/beam/runners/spark/SparkRunnerRegistrar.java

```diff
@@ -41,8 +39,7 @@ private SparkRunnerRegistrar() {}
   public static class Runner implements PipelineRunnerRegistrar {
     @Override
     public Iterable<Class<? extends PipelineRunner<?>>> getPipelineRunners() {
-      return ImmutableList.of(
-          SparkRunner.class, TestSparkRunner.class, SparkStructuredStreamingRunner.class);
+      return ImmutableList.of(SparkRunner.class, TestSparkRunner.class);
```

Comment:
Is it a breaking change? If yes then it would make sense to add a note about that into `CHANGES`

Reply:
No it is not a breaking change because the SparkStructuredStreamingRunnerProvider takes care of registering the runner class independently (and this is validated by the tests (Nexmark concretely) who rely on this shortened translation.